rasa/nlu/selectors/response_selector.py

```diff
@@ -204,6 +205,9 @@ def required_components(cls) -> List[Type[Component]]:
         MASKED_LM: False,
         # Name of the intent for which this response selector is to be trained
         RETRIEVAL_INTENT: None,
+        # Boolean flag to check if actual text of the response
+        # should be used as ground truth label for training the model.
+        TRAIN_ON_TEXT: True,
```

Comment:
Reminder for myself: Change this back to False, before the merge. It is `True` right now for one run of model regression tests.